client/src/components/ExampleProductList.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useGetProductsQuery } from '../reducers/Example-Api-Slice';
+
+const ProductsList = () => {
+  const {
+    data: products,
+    isLoading,
+    isSuccess,
+    isError,
+    error,
+  } = useGetProductsQuery();
+
+  let content;
+
+  if (isLoading) {
+    content = (
+      <p>
```

Comment:
@kyang9021 I'm getting the warning` "Warning: Each child in a list should have a unique "key" prop."` Can you add `<p key={product.id}>` here?

Reply:
Yup, Got it!